### PR TITLE
User timezones

### LINKED ISSUE #1509: Respect user timezone in all dates

### PR DESCRIPTION
**Pull request by abemassry**
closes #1509

### CODE REVIEW

File: src/components/TimeDisplay.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  console.log('timezone', timezone);
+  if (timezone !== null) {
+    return (
+      <span
+        title={moment.utc(props.time, moment.iso_8601).tz(timezone).format('mmm d yyyy h:mm a z')}
+      >
+        {moment.utc(props.time, moment.iso_8601).tz(timezone).fromnow()}
+      </span>
+    );
+  } else {
```
Comment (eatonphil) [inline src/components/TimeDisplay.js]: Could you not repeat this entire section like this? If the timezone field is blank it should be 'UTC' or whatever tz expects.

Comment (abemassry) [inline src/components/TimeDisplay.js]: I'm not sure how to call moment with the .tz and the moment.iso_8601 yet

**Comment by mparke** [comment]
Tested locally, looks good, 

one odd thing we may want to adjust in personal access tokens:

screen shot 2017-04-20 at 10 11 00 am

File: src/components/TimeDisplay.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React, { PropTypes } from 'react';
+import moment from 'moment-timezone';
+
+import { getStorage } from '~/storage';
+
+export default function TimeDisplay(props) {
+  const timezone = getStorage('authentication/timezone') || 'UTC';
+  const utcTime = moment.utc(props.time, moment.iso_8601);
+  utcTime.tz(timezone);
```
Comment (eatonphil) [inline src/components/TimeDisplay.js]: Let's make sure this actually mutates the object, it was just my previous guess.

Comment (mparke) [inline src/components/TimeDisplay.js]: may need `utcTime = utcTime.tz(timezone);`

**Pull request merged**